### PR TITLE
chore(client-app): use @scalar/themes tailwind preset

### PR DESCRIPTION
**Pull request by hwkr**
Also some tailwind cleanup, shouldn't have any visual changes

### CODE REVIEW

File: packages/client-app/src/components/ActionModal/ActionModal.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@ const changeTab = (index: number) => {
     <div
       class="animate-modal-fade bg-backdrop fixed left-0 top-0 z-[1001] h-screen w-screen p-[20px] opacity-0">
       <DialogPanel
-        class="animate-modal-pop before:bg-b-1 relative mx-auto mt-20 w-full max-w-[480px] scale-[0.98] rounded-lg opacity-0 before:absolute before:z-0 before:block before:h-full before:w-full before:rounded-lg before:content-['']">
```
Comment (hwkr) [inline packages/client-app/src/components/ActionModal/ActionModal.vue:56]: @cameronrohani I moved this to a `<div>` at first but it didn't seem to be doing anything so I removed it

File: packages/themes/src/tailwind.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export default {
     borderWidth: {
       'DEFAULT': 'var(--scalar-border-width)',
       '1/2': 'calc(var(--scalar-border-width) / 2)',
+      '0': '0',
```
Comment (antlio) [inline packages/themes/src/tailwind.ts:18]: good catch, make sense since we are not extending

**Review by antlio — APPROVED** [review]
tested out all seems about right, great move!

**Review by amritk — APPROVED** [review]

**Review by marclave — APPROVED** [review]
lets go tailwind, tested!

**Pull request merged**